spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -161,47 +162,93 @@
   end
 
-  scenario "unauthenticated user successfully views list of public adventures" \
-  " by navigating to the all_public adventures page" do
-    # # bucket_list_1
-    # bucket_list_1 = FactoryGirl.create(:bucket_list, user_id: 1)
-    # adventure_1 = FactoryGirl.create(
-    #   :adventure,
-    #   user_id: 1,
-    #   is_shared: true
-    # )
-    # bucket_list_adventure_1 = FactoryGirl.create(
-    #   :bucket_list_adventure,
-    #   bucket_list_id: bucket_list_1.id,
-    #   adventure_id: adventure_1.id
-    # )
-    #
-    # # bucket_list_2
-    # bucket_list_2 = FactoryGirl.create(
-    #   :bucket_list,
-    #   user_id: 2,
-    #   title: "Mongolia",
-    #   is_shared: false
-    # )
-    # adventure_2 = FactoryGirl.create(
-    #   :adventure,
-    #   user_id: user.id,
-    #   name: "Sleep in a yurt"
-    # )
-    # bucket_list_adventure_2 = FactoryGirl.create(
-    #   :bucket_list_adventure,
-    #   bucket_list_id: bucket_list_2.id,
-    #   adventure_id: adventure_2.id
-    # )
-    #
-    # visit bucket_list_path(bucket_list_1.id)
-    #
-    # expect(page).to have_content(bucket_list_1.title)
-    # expect(page).to have_content(adventure_1.name)
-    # expect(page).not_to have_content(bucket_list_2.title)
-    # expect(page).not_to have_content(adventure_2.name)
-  end
+  scenario "authenticated user successfully views list of everyone's public" \
+  " adventures by navigating to the all_public adventures page" do
+    user = FactoryGirl.create(:user)
+    user_b = FactoryGirl.create(:user)
+    sign_in(user_b)
 
+    # bucket_list_public_1
+    bucket_list_public_1 = FactoryGirl.create(
+      :bucket_list,
+      user_id: user.id,
+      is_public: true
+    )
+    adventure_public_1 = FactoryGirl.create(
+      :adventure,
+      user_id: user.id,
+      is_shared: true
+    )
+    bucket_list_adventure_public_1 = FactoryGirl.create(
+      :bucket_list_adventure,
+      bucket_list_id: bucket_list_public_1.id,
+      adventure_id: adventure_public_1.id
+    )
+
+    # bucket_list_public_2
+    bucket_list_public_2 = FactoryGirl.create(
+      :bucket_list,
+      user_id: user.id,
+      title: "Mongolia",
+      is_public: true
+    )
+    adventure_public_2 = FactoryGirl.create(
+      :adventure,
+      user_id: user.id,
+      name: "Sleep in a yurt",
+      is_shared: true
+    )
+    bucket_list_adventure_public_2 = FactoryGirl.create(
+      :bucket_list_adventure,
+      bucket_list_id: bucket_list_public_2.id,
+      adventure_id: adventure_public_2.id
+    )
+
+    # bucket_list_private_3
+    bucket_list_private_3 = FactoryGirl.create(
+      :bucket_list,
+      user_id: user_b.id,
+      title: "Around the world in 80 days",
+      is_public: false
+    )
+    adventure_private_3 = FactoryGirl.create(
+      :adventure,
+      user_id: user_b.id,
+      name: "hot air balloon ride",
+      is_shared: false
+    )
+    bucket_list_adventure_3 = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure_3`. Did you mean `bucket_list_adventure_public_1`?